addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -101,6 +102,20 @@ def _compute_account_type(self):
         for record in self:
             record.account_type = 'income' if sum(record.move_line_ids.mapped('balance')) < 0 else 'expense'
 
+    @api.depends('action', 'move_line_ids')
+    def _compute_lock_date_message(self):
+        for record in self:
+            if record.action == 'change_period':
+                for aml in record.move_line_ids:
+                    lock_date_message = aml.move_id._get_lock_date_message(aml.date, aml.move_id._affect_tax_report())
+                    if lock_date_message:
+                        record.lock_date_message = lock_date_message
+                        break
```

Comment:
The `break` is a bit weird though, we don't really know which date changed for which move

Reply:
Yes. At least it alerts the user that something was changed. :smile:

Reply:
I will check with Benj what he wants in this case.